### PR TITLE
Added Calendar extension

### PR DESCRIPTION
**Pull request by fuksman**
## Description

A simple extension that allows to view calendar month-by-month.

## Type of change
- New extension

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/calendar/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { ActionPanel, CopyToClipboardAction, Detail, getPreferenceValues } from "@raycast/api";
+import { useState, useEffect } from "react";
+import { Calendar } from "calendar";
+
+const days = [
+  "\`SUN\` \`MON\` \`TUE\` \`WED\` \`THU\` \`FRI\` \`SAT\`",
+  "\`MON\` \`TUE\` \`WED\` \`THU\` \`FRI\` \`SAT\` \`SUN\`"
+]
```
Comment (pernielsentikaer) [inline extensions/calendar/src/index.tsx:8]:
```suggestion
const days = [
  "`SUN` `MON` `TUE` `WED` `TEU` `FRI` `SAT`",
  "`MON` `TUE` `WED` `THU` `FRO` `SAT` `SUN`"
]
```

Escaping should not be necessary here.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @fuksman 🚀 

Very nice extension, it has huge potential, and here is some feedback and nice-to-have features:

- Make so the calendar itself uses more of the space available.
- A hotkey to get back to "current month" 
- Add so its possible to see week numbers

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @fuksman 👋 

Thanks for the conversation about this extension in Slack, it was very constructive!

A single thing left:

- Provide icons to every element in the ActionPanel according to [the guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#action-panel) 

When this is corrected everything should be ready to get merged 🥇 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @fuksman 

Thanks for the conversation about the extension, it was all my pleasure 🥇  I think we're ready to merge it 💣  Thanks for your contribution 💯

**Comment by PitNikola** [comment]
CleanShot 2021-11-10 at 14 05 26@2x

Any thoughts on renaming the command to distinguish better from Apple's Calendar app? Not sure what could be a suitable name though... "Quick Calendar", "Monthly Calendar"...

Also few improvement suggestions for the future:
- Indicate current day (may be add bullet symbol at today's date, e.g. `•10`). This would help quickly navigating in current month.
- When running "Current Month" action when it's already presented, it would be nice to show a toast saying something like "Already showing current month". This way user won't be confused that nothing happens (it's not particularly clear). Check our built-in "Reset Raycast Window Position" command for similar behavior.

**Comment by pernielsentikaer** [comment]
We discussed it because the name was "View Calendar" before and that didn't add anything good - Both "Quick" and "Monthly" sound good, I actually like Quick most of them.

Both are awesome ideas we could probably get done before the merge and close version 1 after that

**Comment by fuksman** [comment]
@PitNikola, thanks for your suggestions!
1. Extension name is "Quick Calendar" now
2. Current day is marked with bullet symbol
3. Notification is shown when user continuously sets current month

image

File: extensions/calendar/package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "name": "calendar",
+  "title": "Quick Calendar",
+  "description": "View calendar month-by-month",
+  "icon": "command-icon.png",
+  "author": "fuksman",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Calendar",
```
Comment (PitNikola) [inline extensions/calendar/package.json:12]:
```suggestion
      "title": "Quick Calendar",
```

**Pull request merged**